trlx/model/accelerate_ppo_model.py

```diff
@@ -24,8 +24,21 @@
 @register_model
 class AcceleratePPOModel(AccelerateRLModel):
     def __init__(self, config, train_mode=True):
+        super().__init__(config, train_mode)
+
         self.store = PPORolloutStorage()
-        super().__init__(config, self.store)
+
+        rollout_loader = self.store.create_loader(
+            self.config.train.batch_size, shuffle=True
+        )
+        self.model, self.opt, self.scheduler, rollout_loader = self.accelerator.prepare(
+            self.model, self.opt, self.scheduler, rollout_loader
+        )
+        self.store.clear_history()
```

Comment:
It turns out when passing a dataloader into deepspeed it is required to be nonempty. I had a hack that loaded a dummy prompt and then clears it once things are loaded. Is this still in the code? I cannot seem to find it

Reply:
are you talking about clear_history call?